packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js

```diff
@@ -6,67 +6,98 @@
 /* jshint browser: false, node: true, strict: true */
 'use strict';
 
+const fs = require( 'fs' );
 const path = require( 'path' );
+const glob = require( 'glob' );
+const minimatch = require( 'minimatch' );
+const tmp = require( 'tmp' );
+const karmaLogger = require( 'karma/lib/logger.js' );
 const getWebpackConfigForAutomatedTests = require( './getwebpackconfig' );
 const transformFileOptionToTestGlob = require( '../transformfileoptiontotestglob' );
 
-const reporters = [
+const AVAILABLE_REPORTERS = [
 	'mocha',
 	'dots'
 ];
 
-const coverageDir = path.join( process.cwd(), 'coverage' );
-
 /**
  * @param {Object} options
  * @returns {Object}
```

Comment:
Or `undefined` if the error is logged in this function

Reply:
I'm not sure whether we document it.

Reply:
We document it if a function might return something else.